cmd/buildah/unshare.go

```diff
@@ -181,6 +181,7 @@ func maybeReexecUsingUserNamespace(c *cli.Context, evenForRoot bool) {
 	cmd.UidMappings = uidmap
 	cmd.UseNewgidmap = uidNum != 0
 	cmd.GidMappings = gidmap
+	//fmt.Printf("AAA %s %s\n", c.GlobalIsSet("enable-setgroups"), c.GlobalBool("enable-setgroups"))
```

Comment:
Is this debugging?

Reply:
argh yes, I've added the modified files with `git add` but then I've used `git commit -a` that included also the changes that weren't supposed to be added.

Fixed now